### PR TITLE
Add 'GIGA.de' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22giga%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22name%22%3A%22GIGA.de%22%2C%22address%22%3A%22%E2%84%85%20Str%C3%B6er%20Media%20Brands%20GmbH%5CnTorstra%C3%9Fe%2049%5Cn10119%20Berlin-Mitte%22%2C%22email%22%3A%22datenschutz-giga%40stroeermediabrands.de%22%2C%22web%22%3A%22https%3A%2F%2Fwww.giga.de%22%2C%22sources%22%3A%5B%22https%3A%2F%2Fwww.giga.de%2Finfo%2Fimpressum%22%2C%22https%3A%2F%2Fwww.giga.de%2Fdatenschutz%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1337%22%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22comments%22%3A%5B%22Alternative%20E-Mail-Adresse%3A%20smb-datenschutzbeauftragter%40stroeer.de%22%5D%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

**Comment by WebworkrNet** [comment]
I have opened this issue via [datenanfragen.de](https://www.datenanfragen.de).
You can contact me if you have any questions.

File: companies/giga.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "giga",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "GIGA.de",
+    "address": "℅ Ströer Media Brands GmbH\nTorstraße 49\n10119 Berlin-Mitte",
```
Comment (moriarty-jim) [inline companies/giga.json:7]: I think the correct formatting should be:
`℅ Ströer Media Brands GmbH\nTorstraße 49\n10119 Berlin\nDeutschland`

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
fix CI error

File: companies/giga.json (hunk 1 above)
Comment (baltpeter) [reply companies/giga.json:7]: Yep, exactly. +1

**Review by baltpeter — APPROVED** [review]
Thank you!

**Comment by baltpeter** [comment]
It's somewhat appropriate for Giga to be #1337. :)

**Pull request merged**